### DISCUSSION TITLE
Not getting notification on Android after release

### DISCUSSION THREAD

**Discussion by pk936** [invertase/react-native-firebase#4570] [Q&A] [answered]
I followed all steps mentioned in these 2 pages:  

https://rnfirebase.io/messaging/usage
https://rnfirebase.io/messaging/notifications

I used Firebase console to send notification which worked fine in debug mode- emulator and real device both.

But in release mode I do not get notification in both states - quite or background.

**Comment by mikehardy** [comment, marked as answer]
This sounds like an issue, not a discussion. Our issue template has the information we would need to see in order to help, the information provided here isn't sufficient. For FCM especially please send JSON directly using the REST API and post (minus your API keys and device token!) the actual JSON sent, also please make sure you reproduce with up to date versions there have been underlying bug fixes in the firebase SDKs with regard to messaging and notifications etc, also triple-check your configs and device token etc

**Reply by pk936** [reply]
"@react-native-firebase/admob": "^7.6.12",
    "@react-native-firebase/app": "^9.0.0",
    "@react-native-firebase/messaging": "^8.0.1",


Here is my JSON that finally created by Firebase Console and sent to my app:

```
{
  "notification": {
    "android": {},
    "body": "Here is My Notification text",
    "title": "Notification title"
  },
  "from": "23232321494",
  "ttl": 2419200,
  "sentTime": 1605713787809,
  "data": {
    "Name": "Bill Gates"
  },
  "messageId": "0:1605713787845411%3b84060e3b8d060e",
  "collapseKey": "com.appname"
}
```

**Reply by mikehardy** [reply]
> This sounds like an issue, not a discussion. Our issue template has the information we would need

Please open an issue and we can investigate. Your versions are not up to date though, and note that's the JSON you receive, not the JSON you send
